repos/system_upgrade/common/actors/cloud/checkhybridimage/tests/test_checkhybridimage.py

```diff
@@ -1,12 +1,11 @@
 import pytest
 
+from leapp import reporting
 from leapp.libraries.actor import checkhybridimage
-from leapp.libraries.common.testutils import produce_mocked, create_report_mocked, CurrentActorMocked
+from leapp.libraries.common.testutils import create_report_mocked, CurrentActorMocked, produce_mocked
 from leapp.libraries.stdlib import api
 from leapp.models import FirmwareFacts, InstalledRPM, RPM
```

Comment:
@pirat89 RPM ordering should be fine now, it's case-insensitive lexicographic importing now as you wanted.